### PR TITLE
Added the ability to use a custom path.

### PR DESCRIPTION
**Pull request by Taronyuu**
I was unable to get this to work properly while using the Laravel scheduler because I had to specify the full path to the ping command. (Running OSX) So added a small optional parameter.

### CODE REVIEW

File: JJG/Ping.php [hunk 1]
```diff
@@ -27,37 +27,41 @@ class Ping {
...
-   *     - 128 = same continent
-   *     - 255 = unrestricted
-   *   The TTL is also used as a general 'timeout' value for fsockopen(), so if
-   *   you are using that method, you might want to set a default of 5-10 sec to
-   *   avoid blocking network connections.
-   *
-   * @throws \Exception if the host is not set.
-   */
-  public function __construct($host, $ttl = 255) {
+    /**
```
Comment (geerlingguy) [inline JJG/Ping.php]: Can you leave the indentation as-is? I'd like to clean up formatting to follow the proper PSR in the future, but for now, let's keep changes to the lines that are added only :)

File: JJG/Ping.php [hunk 2]
```diff
@@ -27,37 +27,41 @@ class Ping {
...
+     *     - 1 = same subnet
+     *     - 32 = same site
+     *     - 64 = same region
+     *     - 128 = same continent
+     *     - 255 = unrestricted
+     *   The TTL is also used as a general 'timeout' value for fsockopen(), so if
+     *   you are using that method, you might want to set a default of 5-10 sec to
+     *   avoid blocking network connections.
+     * @param string $path
+     *   Full path to your ping command. (e.g. /sbin/ping on OSX or /bin/ping on Linux)
```
Comment (geerlingguy) [inline JJG/Ping.php]: Can you just leave one path example, like `e.g. /bin/ping on Linux`? That keeps the line < 80 cols and still gets the point across.

File: JJG/Ping.php (hunk 1 above)
Comment (Taronyuu) [inline JJG/Ping.php]: Excuse me, probably done by my editor without me knowing.

File: JJG/Ping.php (hunk 2 above)
Comment (Taronyuu) [inline JJG/Ping.php]: And done :)

**Comment by geerlingguy** [comment]
I think other PRs I've merged conflicted here; I am happy to accept this PR if you can rebase it again—sorry about the delay!

**Pull request closed without merging**